src/Tempest/View/src/Elements/CollectionElement.php

```diff
@@ -5,7 +5,6 @@
 namespace Tempest\View\Elements;
 
 use Tempest\View\Element;
-use Tempest\View\ViewRenderer;
 
 final class CollectionElement implements Element
```

Comment:
We don't really need this `CollectionElement` anymore, just needs a little refactor in `ViewComponentElement` and we can delete it

Reply:
Well maybe it isn't as easy a refactor as I thought. Will keep it for now